frontend/windows/handler.go

```diff
@@ -0,0 +1,23 @@
+package windows
+
+import (
+	"github.com/Azure/dalec/frontend"
+	"github.com/moby/buildkit/frontend/subrequests/targets"
+)
+
+const (
+	targetKey = "windows"
```

Comment:
With the router changes, can we denote the cross-compilation with another path component? The output of this target is still a windows container, so I'd prefer to have it just named `windows`.

I was thinking `windows/cross/container` `windows/cross/zip`. The router will make these code paths pretty easy to follow. WDYT? Am I missing anything here?

Reply:
I don't think this will work because an actual windows build would be totally different, but this is used in the spec directly:

```yaml
targets:
  windows:
    # windows specific stuff
```

The main thing there is build dependencies will be different.
Looking through what we have, that may be the only thing that's particularly problematic in a world where the targets are shared.

Perhaps the target key could be the path? In which case I'd do something like `jammy/cross/windows` or something to denote it's the jammy toolchain.
I'm not sure if this works or not so maybe worth testing?

Reply:
I guess this has always been ambiguous up until now. Does the first component of the path denote the build distro/OS or the output distro/OS?

We haven't had to decide this yes since the build & output distros/OSes have always been the same.

Reply:
It is indeed a combination of build toolchain and output.

Reply:
As we discussed, I'm changing the target name to "windowscross". If there is a routing solution that disambiguates build toolchain from output, we'll include those changes in #172 (the routing PR)

Reply:
this is done